### PR TITLE
Add images to the Block step page

### PR DESCRIPTION
**Pull request by clbarrell**
Added updated images using template file from Figma to the page. 

Also removed one unnecessary screenshot from bottom of page.

### CODE REVIEW

File: images/docs/pipelines/block_step/block-step-confirm-modal.png
Comment (mbelton-buildkite) [inline images/docs/pipelines/block_step/block-step-confirm-modal.png:1]: Did you take these on your retina screen rather than your external monitor?

File: images/docs/pipelines/block_step/block-step-button.png
Comment (mbelton-buildkite) [inline images/docs/pipelines/block_step/block-step-button.png:1]: For these and the other images, it would be great to avoid including any personal details. Thinking:

- Replace your name with `Sam Kim`.
- Replace the agent name with `agent.local-1`

If you're comfortable with it and don't need to retake them for other reasons, you can leave these ones.

File: images/docs/pipelines/block_step/block-step-text-field.png
Comment (mbelton-buildkite) [inline images/docs/pipelines/block_step/block-step-text-field.png:1]: Can you please include removing the old images in this change?

File: pages/pipelines/block_step.md [hunk 1]
```diff
@@ -412,8 +414,6 @@ steps:
 
 The command step added in the above example will upload the trigger step and add it to the end of our pipeline at runtime.
```
Comment (mbelton-buildkite) [inline pages/pipelines/block_step.md:415]: A style fix while we're here:

```suggestion
The command step added in the above example uploads the trigger step and adds it to the end of the pipeline at runtime.
```

File: pages/pipelines/block_step.md [hunk 2]
```diff
@@ -412,8 +414,6 @@ steps:
...
 
-<%= image "block-trigger-pipeline.png", width: 2328/2, height: 952/2, alt: "Screenshot of pipeline showing the uploaded trigger step" %>
-
 In the pipeline you're triggering, you will be able to use the meta-data that you have passed through as if it was set during the triggered build.
```
Comment (mbelton-buildkite) [inline pages/pipelines/block_step.md:417]: Another style fix while we're here:

```suggestion
In the pipeline you're triggering, you can use the meta-data passed through as if it was set during the triggered build.
```

File: pages/pipelines/block_step.md [hunk 3]
```diff
@@ -412,8 +414,6 @@ steps:
 
 The command step added in the above example will upload the trigger step and add it to the end of our pipeline at runtime.
 
-<%= image "block-trigger-pipeline.png", width: 2328/2, height: 952/2, alt: "Screenshot of pipeline showing the uploaded trigger step" %>
```
Comment (mbelton-buildkite) [inline pages/pipelines/block_step.md:415]: Also remove the corresponding file from the repo.

File: pages/pipelines/block_step.md [hunk 4]
```diff
@@ -11,20 +11,22 @@ Once all steps before the block have completed, the pipeline will pause and wait
 
 ```yml
 steps:
-  - block: "\:rocket\: Release!"
+  - block: "\:rocket\: Are we ready?"
 ```
 {: codeblock-file="pipeline.yml"}
 
-<%= image "block-step.png", width: 944/2, height: 364/2, alt: "Screenshot of a basic block step" %>
+<%= image "block-step-button.png", alt: "Screenshot of button to press to unblock a block step" %>
```
Comment (dannymidnight) [inline pages/pipelines/block_step.md:18]: As mentioned offline, I was thinking that we might need the widths + heights for these images but now that I see them full width I think it's probably OK – assuming the image is at least double the container column size.

Comment (clbarrell) [reply pages/pipelines/block_step.md:18]: Yeah they are exported at 2x 1080

**Comment by clbarrell** [comment]
FYI I have made the images on Retina screen and followed all recc's

**Comment by olyism** [comment]
Hi @clbarrell I resolved the errors by rebasing from `main` and forced pushed. Everything should be fine but please double check that the content looks good in case I broke anything (I've also got a backup locally too).

**Review by olyism — APPROVED** [review]
LGTM 👍 Great work!

File: pages/agent/v3/elastic_ci_aws.md
Comment (mbelton-buildkite) [inline pages/agent/v3/elastic_ci_aws.md:1]: This looks like it's not part of the change. Is it a hangover from Git issues?
